content/library/api/performance/experimental-singleton.md

```diff
@@ -43,8 +43,6 @@ The `@st.experimental_singleton` decorator is used to cache the output of a func
 
 However, in some cases, the cached output may become invalid over time, such as when a database connection times out. To handle this, the `@st.experimental_singleton` decorator supports an optional `validate` parameter, which accepts a validation function that is called each time the cached output is accessed. If the validation function returns False, the cached output is discarded and the decorated function is executed again.
 
-<!-- Let's learn how to use the `validate` parameter to ensure that cached output remains valid. Let's also look at specific examples and best practices to help you understand how to use this feature effectively.
```

Comment:
I think something got mixed up on this page. An opening comment from line 46 looks removed but the commented section ends (originally) on 259. There are some quadruple backticks introduced in the middle, but I think the whole section should be removed to take out the outdated content.

Reply:
Good catch! Removed.